js/main.js

```diff
@@ -31,3 +31,47 @@ const appendElementWithText = function (parent, tagName, text) {
   element.appendChild(textNode)
   parent.appendChild(element)
 }
+
+const buildRecipeDetails = function (recipeDetails) {
+  const dl = document.createElement('dl')
+
+  for (let i = 0; i < recipeDetails.length; i++) {
```

Comment:
You're using `for` a lot, and it's working great (in fact, it has been working great for a long time in the JavaScript world). Javascript now offers a lot of nice, functional methods working on collections. In this particular case, you're causing side-effects. I googled this article which seems like a good read: https://medium.com/@ntgard/foreach-is-for-side-effects-60fca9f78850 - can you find a better suited function you could use here instead of a `for`?